neo/Consensus/ConsensusService.cs

```diff
@@ -558,7 +556,7 @@ private void RequestChangeView()
             if ((context.CountCommitted() + context.CountFailed()) > context.F())
             {
                 Log($"Skip requesting change view to nv={expectedView} because nc={context.CountCommitted()} nf={context.CountFailed()}");
-                SendChangeViewToRequestRecovery(context.ViewNumber);
+                localNode.Tell(new LocalNode.SendDirectly { Inventory = context.MakeChangeView(context.ViewNumber) });
```

Comment:
this _looks_ better this way... so that OnChangeView event only sends ChangeView message.
but perhaps it's better to `context.MakeRecoveryRequest()` here instead of MakeChangeView.

Reply:
What do you think @vncoelho ?

Reply:
I agree with you, brother.